packages/contracts/src/dollar/DirectGovernanceFarmer.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity >= 0.8.3;
+pragma solidity >=0.8.3;
```

Comment:
I believe we're using solidity 0.8.16 now.

Reply:
done